pkg/kustomize/interface.go

```diff
@@ -3,4 +3,7 @@ package kustomize
 // Kustomizer defines common kustomize actions used within Jenkins X
 type Kustomizer interface {
 	Version(extraArgs ...string) (string, error)
+	HasKustomize() bool
+	FindKustomize() (resource []string)
+	SetCWD(dir string)
```

Comment:
What is the aim of _SetCWD_? It seems odd to see this in the Kustomizer interface.